sympy/ntheory/tests/test_generate.py

```diff
@@ -107,7 +108,23 @@ def test_generate():
     assert nextprime(10**40) == (10**40 + 121)
     assert prevprime(97) == 89
     assert prevprime(10**40) == (10**40 - 17)
+
     assert list(sieve.primerange(10, 1)) == []
+
+    assert list(sieve.totientrange(5, 15)) == [4, 2, 6, 4, 6, 4, 10, 4, 12, 6]
+    sieve._tlist = _array('l', [0, 1, 1, 2, 2, 4])
```

Comment:
I guess it would have made more sense to use the reset. You had that in the commit title. Did you forget to put it in?

Reply:
Yes, actually `_reset` would reset all 3 lists `_list`, `_tlist`, `_mlist`. Should I use it?

Reply:
Looks good to me. Should mobius use array of `ints` for memory saving (2 bytes vs 4 bytes per element)?